gcc-checker-plugin-toolfile.spec

```diff
@@ -0,0 +1,43 @@
+### RPM cms gcc-checker-plugin-toolfile 1.0
+Requires: gcc gcc-checker-plugin
+
+%prep
+%build
+%install
+mkdir -p %{i}/etc/scram.d
+
+cat << \EOF_TOOLFILE >%i/etc/scram.d/gcc-checker-plugin.xml
+  <tool name="gcc-checker-plugin" version="@TOOL_VERSION@">
+    <client>
+      <environment name="GCC_CHECKER_PLUGIN_ROOT" default="@TOOL_ROOT@"/>
+      <environment name="INCLUDE"   default="$GCC_CHECKER_PLUGIN_ROOT/include"/>
+    </client>
+  </tool>
+  <runtime name="GCC_CHECKER_PLUGIN" default="$GCC_CHECKER_PLUGIN_ROOT/lib/libchecker_gccplugins.so"/>
```

Comment:
do we need GCC_CHECKER_PLUGIN env variable?

Reply:
It is no longer needed.